src/main/asciidoc/java/http.adoc

```diff
@@ -1538,8 +1538,24 @@ When pipe-lining is enabled requests will be written to connections without wait
 
 === HTTP/2 multiplexing
 
-For HTTP/2, the http client uses a single connection for each server, all the requests to the same server are
-multiplexed on the same connection.
+HTTP/2 advocates to use a single connection to a server, by default the http client uses a single
+connection for each server, all the streams to the same server are multiplexed on the same connection.
+
+When it is desirable to limit the number of concurrent streams per server and uses a connection
+pool instead of a single connection, `link:../../apidocs/io/vertx/core/http/HttpClientOptions.html#setMaxStreams-int-[setMaxStreams]`
+can be used.
+
+[source,java]
+----
+HttpClientOptions clientOptions = new HttpClientOptions().setMaxStreams(10).setMaxPoolSize(3);
+
+// Uses up to 3 connections and up to 10 streams per connection
+HttpClient client = vertx.createHttpClient(clientOptions);
+----
+
+The maximum streams for a connection is a setting set on the client that limits the streams
+of a single connection. The effective value can be even lower if the server sets a lower limit
```

Comment:
"The maximum streams for a connection is a setting set on the client that limits the streams of a single connection." 

should be

"`setMaxStreams` is an option set on the client limiting the number of streams for a single connection."

(Better English)